requirements/test_redirects.txt

```diff
@@ -0,0 +1,9 @@
+-r dev.txt
```

Comment:
That's correct, although I suspect at some point we may want to include the redirect tests.